### PR TITLE
Removes Programmable Unloaders and a Few Unused Files

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Removes a few unused files.

Also removes programmable unloaders. No on uses these, ever, the code is absolutely atrocious, it's unmaintaiend, and not really needed anymore.

:cl: Fox McCloud
del: Removed programmable unloader circuit
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/syndie_uplink.dm [hunk 1]
```diff
@@ -1,38 +0,0 @@
...
-	var/obj/item/radio/origradio = null
-	flags = CONDUCT | ONBELT
-	w_class = WEIGHT_CLASS_SMALL
-	item_state = "radio"
-	throw_speed = 4
-	throw_range = 20
-	materials = list(MAT_METAL=100)
-	origin_tech = "magnets=2;syndicate=3"*/
-
-/obj/item/SWF_uplink
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/syndie_uplink.dm]: I bet this is 4407 stuff.

File: code/game/objects/items/weapons/wires.dm [hunk 2]
```diff
@@ -1,36 +0,0 @@
-// WIRES
-
-/obj/item/wire
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/wires.dm]: Why wasn't this removed when cable become a stack? Why? WHY?!

**Comment by taukausanake** [comment]
What is a programmable unloader? I've never heard of such a thing

**Comment by alexkar598** [comment]
I kinda like now and I want to try it

**Comment by Fox-McCloud** [comment]
`What is a programmable unloader? I've never heard of such a thing`

Some thing from VG that was added on a lark and barely used. It's supposed to be some platform for sorting objects automatically. I'm guessing it's hypothetically supposed to be used in some ridiculously overcomplicated cargo setup like Goon has (ie: belt hell).

**Comment by alexkar598** [comment]
Well,it sounds cool but I'm pretty sure that its completely broken rn so it's probably a better idea to remove it and reimplement it latter

**Comment by Pennwick** [comment]
Actually I like using the programmable unloader when I fully set up recycling. I set it up on the conveyor in the mail room to filter the stacks of metal and glass while almost everything else passes through.

**Comment by taukausanake** [comment]
@Pennwick does that mean it works? I'd like to see what it does exactly, sounds interesting especially if like Goon mail sorting I'm thinking of

**Comment by tigercat2000** [comment]
If we want programmable unloaders they need to be remade from scratch with not shitty broken code

**Pull request merged**